src/components/toolbar.tsx

```diff
@@ -0,0 +1,94 @@
+import React , {useState, useEffect} from 'react';
+import '../css/toolbar.scss';
+import {Button , Box , Select, Option} from '@mui/joy';
+import IconButton from '@mui/joy/IconButton';
+import SearchIcon from '@mui/icons-material/Search';
+import MenuIcon from '@mui/icons-material/Menu';
+import AddCircleOutlinedIcon from '@mui/icons-material/AddCircleOutlined';
+
+
+
+
+function Toolbar() {
+  const [view,setView] = useState('Day')
+  const [showTime,setShowTime] = useState('')
+  const date = '01'
+  const month= 'January'
+  const year='2022'
+  const week='01-07'
+
+  useEffect(()=> {
+    console.log(view)
+    switch(view) {
+      case 'Day':
+        setShowTime([date,month,year].join(' '))
+        break;
+      case 'Week':
+        setShowTime([week,month,year].join(' '))
+        break;
+      case 'Month':
+        setShowTime([month,year].join(' '))
+      break;
+      case 'Year':
+        setShowTime(year);
+      break;
+      default:
+        
+    } 
+
+  },[view])
+
+  const handleChange = (event:any) => {
+    const value = event.target.innerText
+    console.log(event)
+    setView(value);
+    console.log(view);
+  };
+
+
+  return (
+    <div className='toolbar'>
+       <Box sx={{ display: 'flex', gap: 2, alignItems: 'center', justifyContent: 'space-between' }}>
+        <div className='left-content'>
+          <IconButton aria-label="Menu" component="a" variant="plain" color="neutral">
+              <MenuIcon />
+            </IconButton>
+          <h1>{showTime}</h1>
+          <div  className='select-view-component'>    
+            <Select defaultValue="Day" variant='soft' value={view} onChange={handleChange} size="sm">
```

Comment:
Component is intended to be re-used. Toolbar needs to be constructed so that `prop` like `Select` or `Button` can be passed into instead of hard-coded.

Reply:
Could you suggest how to correct it or giving any kinds of example? Thanks